src/Lykke.Service.OperationsHistory.Job/RabbitSubscribers/MongoDbSubscriber.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Autofac;
+using Common;
+using Common.Log;
+using Lykke.RabbitMqBroker;
+using Lykke.RabbitMqBroker.Subscriber;
+using Lykke.Service.ClientAccount.Client;
+using Lykke.Service.ClientAccount.Client.AutorestClient;
+using Lykke.Service.ClientAccount.Client.AutorestClient.Models;
+using Lykke.Service.OperationsHistory.Core;
+using Lykke.Service.OperationsHistory.Core.Services;
+using Lykke.Service.OperationsHistory.Core.Settings.Job;
+using Lykke.Service.OperationsHistory.Mongo;
+using Lykke.Service.OperationsRepository.Contract.History;
+using Microsoft.Extensions.Caching.Distributed;
+using Lykke.Service.OperationsRepository.Contract;
+
+namespace Lykke.Service.OperationsHistory.Job.RabbitSubscribers
+{
+    public class MongoDbSubscriber : IStartable, IStopable
```

Comment:
It looks like more "HistoryConsumer" to me. Anyway it has nothing to do with Mongo.

Reply:
This queue handler will be removed in the next release.